rpcs3/rpcs3qt/emu_settings.h

```diff
@@ -197,6 +198,7 @@ public Q_SLOTS:
 		{ PPUDebug,                 { "Core", "PPU Debug"}},
 		{ SPUDebug,                 { "Core", "SPU Debug"}},
 		{ MaxLLVMThreads,           { "Core", "Max LLVM Compile Threads"}},
+		{ EnableTSX,				{ "Core", "Enable TSX on Haswell/Broadwell"}},
```

Comment:
use spaces instead of tabs here. the indentation is broken.

Reply:
In Visual Studio it looked good - I'll fix that.